fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanOfficerAssignmentHistory.java

```diff
@@ -72,16 +73,16 @@ public void updateEndDate(final LocalDate endDate) {
         this.endDate = endDate;
     }
 
-    public boolean matchesStartDateOf(final LocalDate matchingDate) {
```

Comment:
Arent these changes irrelevant to [FINERACT-1921](https://issues.apache.org/jira/browse/FINERACT-1921)? Please extract them into a separate PR if so!